src/script/telemetry/calling/CallTelemetry.ts

```diff
@@ -40,34 +54,11 @@ class CallTelemetry {
     this.mediaType = MediaType.AUDIO;
   }
 
-  //##############################################################################
-  // Sessions
-  //##############################################################################
-
-  /**
-   * Force log last call session IDs.
-   * @returns {Object} Containing all the sessions
-   */
-  log_sessions() {
```

Comment:
This was not in use and I couldn't figure out how the `sessions` object was supposed to look like.

Reply:
Could it be that QA was using it?

Reply:
Seems like not.